src/Microsoft.OData.Core/UriParser/Aggregation/ApplyBinder.cs

```diff
@@ -116,6 +129,7 @@ private IEdmTypeReference CreateAggregateExpressionTypeReference(SingleValueNode
                                     expressionPrimitiveKind));
                     }
 
+                case AggregationMethod.VirtualPropertyCount:

```

Comment:
According to above code, it seems that this line will never be touched, right? If the token is a virtualPropertyCount, the code will always go to line 82 instead of line 87.